app/views/connections/_connection.html.haml

```diff
@@ -0,0 +1,11 @@
+- index = connection_counter + 1
+%tr(class="connection_#{index}")
+  %td
+    .pull-left
+      %img(src="#{gravatar_image_string(connection.guide)}")
```

Comment:
I would recommend to use `= image_tag gravatar_image_string(connection.guide` to stay consistent with Rails code. It reads better too.